tests/conftest.py

```diff
@@ -255,6 +257,11 @@ def task_mock(*args, **kwargs):
     monkeypatch.setattr(tasks, 'parse_logs', task_mock)
 
 
+@pytest.fixture(autouse=True)
+def mock_taskcluster_notify(monkeypatch):
```

Comment:
I don't like the fact that we run this fixture for *all* our tests. We need to run it only for the Perfherder cycling tests.